src/java/htsjdk/variant/vcf/VCFHeaderLineTranslator.java

```diff
@@ -83,13 +83,29 @@
         // are we inside a quotation? we don't special case ',' then
         boolean inQuote = false;
 
+        // if we are in a quote and we see a backslash followed by quote, treat it as an escaped quote
+        boolean escape = false;
+
         // a little switch machine to parse out the tags. Regex ended up being really complicated and ugly [yes, but this machine is getting ugly now... MAD]
         for (char c: valueLine.toCharArray()) {
-            if ( c == '\"' ) {
-                inQuote = ! inQuote;
+            if ( c == '\"') {
+                if (escape) {
+                    builder.append(c);
+                    escape = false;
+                } else {
+                    inQuote = !inQuote;
+                }
             } else if ( inQuote ) {
-                builder.append(c);
+                if (escape) {
+                    builder.append(c);
+                    escape = false;
+                } else if (c != '\\') {
+                    builder.append(c);
+                } else {
+                    escape = true;
+                }
             } else {
+                escape = false;
                 switch (c) {
                     case ('<') : if (index == 0) break; // if we see a open bracket at the beginning, ignore it
                     case ('>') : if (index == valueLine.length()-1) ret.put(key,builder.toString().trim()); break; // if we see a close bracket, and we're at the end, add an entry to our list
```

Comment:
Actually the spec says that the only supported escape sequences are `\\` and `\"`, but your code would strip the backslash in other cases as well.